### ISSUE TITLE
Incorrect project name inside Podfile

### ISSUE THREAD

**Issue by alanhughes** [GoogleCloudPlatform/fda-mystudies#3566]
**Describe the bug**
This is somewhere between a bug and feature request - at present, if we want to make any changes to Pods or Pod structure (e.g. add new Pods or changing the scheme names) we see this message:
```
[!] Unable to find the Xcode project `/path/to/repo/iOS/MyStudies/HPHC.xcodeproj` for the target `Pods-MyStudies`.
```

**To Reproduce**
1. Delete .xcworkspace, Pods and Podfile.lock
2. Run `pod install`

**Expected behavior**
Fresh `pod install` completes successfully

**Desktop (please complete the following information):**
 - OS: macOS
 - Version 11.7

**Additional context**
Changing [this line](https://github.com/GoogleCloudPlatform/fda-mystudies/blob/v2.0.4/iOS/MyStudies/Podfile#L1) to 
```
project 'MyStudies.xcodeproj/'
```
fixes the issue. This also renames [this folder](https://github.com/GoogleCloudPlatform/fda-mystudies/tree/v2.0.4/iOS/MyStudies/Pods/Target%20Support%20Files/Pods-HPHC) and all files inside it to use MyStudies instead of HPHC as well.

**Labels**
iOS

**Comment by GovindPrasadBostonTechnologyCorporation** [comment]
Rename of the podfile is already addressed in the current v2.0.5 release. The folder structure issue will be addressed in the next release.

**Comment by nishit-v** [comment]
Handled as part of 2.0.7 release
